src/modular/monty_form.rs

```diff
@@ -90,12 +94,18 @@ impl<const LIMBS: usize> MontyParams<LIMBS> {
 
         // The modular inverse should always exist, because it was ensured odd above, which also ensures it's non-zero
         let inv_mod = modulus
-            .inv_mod2k_vartime(Word::BITS)
+            .as_ref()
+            .inv_mod2k_full_vartime(Word::BITS)
             .expect("modular inverse should exist");
 
         let mod_neg_inv = Limb(Word::MIN.wrapping_sub(inv_mod.limbs[0].0));
 
-        let mod_leading_zeros = modulus.as_ref().leading_zeros_vartime().min(Word::BITS - 1);
+        let mod_leading_zeros = modulus.as_ref().leading_zeros_vartime();
+        let mod_leading_zeros = if mod_leading_zeros < Word::BITS - 1 {
+            mod_leading_zeros
+        } else {
+            Word::BITS - 1
+        };
```

Comment:
What was wrong with the old version?

Reply:
`min()` is not `const fn`. Technically, I can revert making these constructors `const` since it is not required by this PR - I had an idea to merge them with `const_monty_form`, but it can wait.

Reply:
Aah, it's `Ord::min`, I see.

I think it's fine to constify this.